app/presenters/notices/setup/prepare-return-forms.presenter.js

```diff
@@ -104,6 +134,37 @@ function _formatPeriodsToLongDate(periods) {
   })
 }
 
+/**
+ * Group an array of dates by the date's month and year.
+ *
+ * The group key (period) will be the month and the year of the date 'June 2024'.
+ *
+ * The date will be added to the 'days' array for the group as the day only (no leading 0).
+ *
+ * @private
+ */
+function _groupDaysIntoMonths(dates) {
+  return dates.reduce((acc, date) => {
+    const key = date.toLocaleString('default', { month: 'long', year: 'numeric' })
+
+    let group = acc.find((group) => {
+      return group.period === key
+    })
```

Comment:
SonarQube is complaining because `group` has been declared twice within this function. Here is my suggested quick fix.

```suggestion
    let group = acc.find((existingGroup) => {
      return existingGroup.period === key
    })
```